code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1336,4 +1336,151 @@
+		M.Dizzy(jitter)
+		M.slurring = slur // End of base probability effects
+	if(heal_roll < 0) // Healing payload after calculations
+		if (M.stat == DEAD) // Non OD , if dead , 30% chance to discard healing payload
+			if(prob(70))
+				M.adjustFireLoss(heal_roll*REM, FALSE, FALSE, BODYPART_ORGANIC)
+				M.adjustBruteLoss(heal_roll*REM, 0)
+		else
+			M.adjustFireLoss(heal_roll*REM, 0) 
+			M.adjustBruteLoss(heal_roll*REM, 0) 
+	..()
+	. = 1
+
+/datum/reagent/drug/burnmix/overdose_start(mob/living/M)
+	to_chat(M, "<span class='userdanger'>You feel the BurnMix increase in potency.</span>")
+
+
+/datum/reagent/medicine/burnmix/overdose_process(mob/living/M)
+	var/heal_roll = pick(-0.2,-0.4)
+	var/slur = 20
+	var/jitter = 2
+	if(M.health <= M.crit_threshold)
+		heal_roll = heal_roll + 0.2 // Burnmix in overdose will heal inside of crit without epipen or perfluorodecalin. Being in crit means a roll for either no base healing or half reduction to base heal.
+	if(heal_roll > 0 && prob(10))
+		heal_roll = 0 // Burnmix has a chance not to basic heal at all
+	if(holder.has_reagent(/datum/reagent/medicine/epinephrine))
+		heal_roll = heal_roll - 0.35
+		jitter = jitter + 15
+		slur = slur + 25
+		if(prob(25))
+			M.losebreath++
+			M.adjustStaminaLoss(2.5*REM, 0)
+			if(prob(80))
+				M.adjustToxLoss(0.5*REM, 0)
+				M.adjustToxLoss(pick(0,-0.25), 0) // gives a chance to negate half epinephrine tox damage
+
+	if(holder.has_reagent(/datum/reagent/medicine/perfluorodecalin))
+		heal_roll = heal_roll - 0.4
+		jitter = jitter + 15
+		slur = slur + 25
+		if(prob(25))
+			M.adjustToxLoss(0.5*REM, 0)
+			M.adjustToxLoss(pick(0,-0.25), 0) // gives a chance to negate half perfluorodecalin tox damage
+
+	if(holder.has_reagent(holder.has_reagent(/datum/reagent/medicine/epinephrine) && /datum/reagent/medicine/perfluorodecalin)) // start IF both drugs present
+		if(prob(0.5)) // Should be 1 in 200 chance of a 1 in 3 chance to roll for faint if both epinephrine and perfluorodecalin are present
+			M.emote((pick("scream","faint","gasp")))
+		if(prob(60)) // Original edit did too much brain damage. Moved to both drug effects. Was default OD effect
+			M.adjustOrganLoss(ORGAN_SLOT_BRAIN,pick(1,-1))
+		if(prob(30))
+			slur = slur + 25
+		if(prob(30))
+			jitter = 50
+		if(prob(30))
+			M.drowsyness += 0.5
+			if(prob(2))
+				to_chat(M, "<span class ='notice'>Your fingers spasm!</span>")
+				M.drop_all_held_items() // end IF both drugs present
+
+	if(prob(40)) // start last set of OD probability effects
+		M.emote(pick("twitch","drool","moan","giggle","spin"))
+		if(prob(60))
+			M.adjustOrganLoss(ORGAN_SLOT_BRAIN, pick(0,1,2,2.5,-1,-2))
+		if(prob(15))
+			M.set_drugginess(pick(2,4,6))
+		M.Jitter(jitter)
+	if(prob(40))	
+		M.adjustToxLoss(pick(1,-1)*REM, 0)
+	if(prob(1))
+		M.Dizzy(jitter) // end last set of OD probability effects
+	M.Jitter(jitter) // OD slur and jitter after calculations
+	M.slurring = slur
+	if(heal_roll < 0) // OD Healing payload after calculations
```

Comment:
```suggestion
	if(heal_roll != 0) // OD Healing payload after calculations
```

Reply:
Hello Alex,
The point of me making that value less than 0 is that I want to ensure that the healing payload only activates when there is healing to be done. When the value goes above 0 I do not want this statement to activate. I am requesting you deny this change.